### ISSUE TITLE
clap plugin support?

### ISSUE THREAD

**Issue by RustoMCSpit** [publicsamples/Oi-Grandad#42]

**Comment by RickTaylor79** [comment]
I'll look into it. If only 'cos a blog post titled "Grandad gets the clap" would be amusing. 

If this is something other people would like to see please speak up, offer assistance, justifications etc.

**Comment by RickTaylor79** [comment]
Yeah, the only reference to clap I can find on the forums basically says they'll support it when JUCE does.

This might offer a solution..

https://github.com/free-audio/clap-juce-extensions

**Comment by RustoMCSpit** [comment]
i know the dev of that, join here https://discord.com/invite/spGANHw

**Comment by RustoMCSpit** [comment]
> hise doesnt seem to support it

are there any other programs like hise? i know of https://github.com/brummer10/XUiDesigner but i dont know anything else?
